packages/openactive-integration-tests/test/helpers/flow-stages/flow-stage-recipes.js

```diff
@@ -712,6 +715,14 @@ const FlowStageRecipes = {
             ...defaultFlowStageParams,
             prerequisite,
             testName: `Orders Feed (after ${cancelTestName})`,
+            /* This allows us to support Booking Systems which update OrderItem
+            statuses one at a time, rather than all at once.
+            (The Seller Requested Cancellation action is expected to cancel all
+            OrderItems)
+            Using ListenerItemRequirements, an update which does not have all
+            OrderItem statuses changed will be ignored (if the config option is
+            enabled). */
+            listenerItemRequirements: [ListenerItemRequirementRecipes.allNonConfirmedOrderItems()],
```

Comment:
This may also be a little confusing as "requirements" as a specific meaning in specs. Would `ListenerItemExpectations` or `ListenerItemConditions` be clearer?